metadata/cs/2020/ColorimetryCS.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ClassificationScheme uri="urn:dvb:metadata:cs:ColorimetryCS:2020" xmlns="urn:tva:mpeg7:2008" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:tva:mpeg7:2008 tva_mpeg7.xsd" xsi:type="ClassificationSchemeType">
+	<!-- ##################################################################### -->
+	<!-- Colorimetry                                                           -->
+	<!-- Definition: Different picture colorimetry standards                   -->
+	<!-- ##################################################################### -->
+	<Term termID="1">
+		<Name xml:lang="en">ITU BT.709 colorimetry</Name>
```

Comment:
The CS is already published. We could add a description in a future update but the name is generally descriptive enough in the video domain, otherwise we would have
````
<Term termid="1">
  <Name>BT.709</Name>
  <Definition xml:lang="en>ITU BT.709 colorimetry</Definition>
</Term>
````

Reply:
> I see. Perhaps a new Bugzilla for the next revision then? 😉
> 
done - [Bug 3002](https://bugzilla.dvb.org/show_bug.cgi?id=3002)